.dev_scripts/github/update_copyright.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# Copyright (c) OpenMMLab. All rights reserved.
+
+import argparse
+import os
+import os.path as osp
+import re
+import sys
+
+HEADER = 'Copyright (c) OpenMMLab. All rights reserved.\n'
+HEADER_KEYWORDS = {'Copyright', 'License'}
+
+
+def contains_header(lines, comment_symbol):
+    for line in lines:
+        if line.startswith('#!'):
+            # skip shebang line
+            continue
+        elif re.match(f'{comment_symbol}.*({"|".join(HEADER_KEYWORDS)})',
```

Comment:
In some comments, we might added licence info of reference codes/papers. Will that fail this check?

Maybe checking for the first few (2?) lines is good enough?

Reply:
Ok

Reply:
fixed.